rfcs/012-image-folder.md

```diff
@@ -0,0 +1,42 @@
+# RFC 012 - Keep Images in a Centralized Location
+
+# Affected projects
+
+- Frontend
+
+# Motivation
+
+As discussed in this issue, https://github.com/opencollective/opencollective-frontend/pull/6158#pullrequestreview-640718161,
+we tend to keep images in any folder. So for example we have image files in components folder;
+
+```
+./components/collective-page/images/ContributorsGridBackground.svg
+./components/collective-page/images/HeroBackgroundMask.svg
+./components/collective-page/images/EmptyCollectivesSectionImage.svg
+./components/icons/PrivateLockIcon.png
+./components/tier-page/Bubbles.svg
+./components/contribution-flow/fees-on-top-illustration.png
+./components/contribution-flow/success-illustration.jpg
+```
+
+It's best to avoid putting images in folders such as `components` which is normally the place for React components,
```

Comment:
Putting images in the components folder is actually a widespread practice. For example, it's the way recommended by create-react-app: https://create-react-app.dev/docs/adding-images-fonts-and-files/